coremltools/converters/mil/mil/passes/tests/test_passes.py

```diff
@@ -7662,6 +7662,109 @@ def prog(x):
             prog, {"x": shape}, expected_output_shapes={block.outputs[0].name: shape}
         )
 
+    @pytest.mark.parametrize("with_affine", [True, False])
+    def test_ane_layer_norm(self, with_affine):
```

Comment:
can you also add some "negative" tests as well? In which the pattern will not be fused.
For instance:
1. some invalid config in the op
2. some ops are connect to the output

Reply:
Sure thing. I added those two tests plus a test for the case where there is an ambiguous "add" at the end.